### PR TITLE
Deprecated.Text.Sexp: fix Lua array index base in 8.2.1066 0 to 1

### PR DESCRIPTION
**Pull request by tsuyoshicho**
Fix lua index base diff.

Fix or remove? (It is deprecated group).

### CODE REVIEW

File: autoload/vital/__vital__/Deprecated/Text/Sexp.vim [hunk 1]
```diff
@@ -28,9 +28,20 @@ function! s:_vital_depends() abort
   \ }
 endfunction
 
+" lua array index as 0 based.
+let s:_base = 0
+if has('patch-8.2.1066')
```
Comment (prabirshrestha) [inline autoload/vital/__vital__/Deprecated/Text/Sexp.vim:33]: Or neovim

**Comment by lambdalisue** [comment]
+1 for remove while it's in `Deprecated`

**Comment by kuuote** [comment]
I agree with you on the removal, because that's deprecated.

**Comment by tsuyoshicho** [comment]
Thanks all.

I think, master CI test fail recovering choices are below:

- Fix this PR.
- Remove this module.
- Fix this PR for temp., and after a while remove it.(another PR)

Currently, removal get 2 vote.

File: autoload/vital/__vital__/Deprecated/Text/Sexp.vim (hunk 1 above)
Comment (tsuyoshicho) [reply autoload/vital/__vital__/Deprecated/Text/Sexp.vim:33]: Thanks, please wait a little.

**Comment by ujihisa** [comment]
Let's remove after merging this to stop bleeding. It provides a good reference for future. Even though it's not useful directly for plugin developers to depend, it can be still handy for them to refer.

**Comment by tsuyoshicho** [comment]
Okay, thanks.

**Pull request merged**